src/api/tests/test_semester.py

```diff
@@ -0,0 +1,12 @@
+
+from .util import Client
+
+    #semester should get all of the semesters in the upload
+    #in the case of our test data, we should be getting Summer and Spring semester of 2020
+def test_semester(upload, client):
+    r = client.get("/api/semester")
```

Comment:
Please move test descriptions inside the function and format with
'''
<Comments>
'''
rather than '#'
Also make your descriptions a little more precise, what endpoint is being tested, what are the inputs and expected outputs? This will be helpful for future developers who might need to work on this API